### PR TITLE
Added preloader

### PR DESCRIPTION
**Pull request by aswanthkoleri**
check it out : [view new loader](https://aswanthkoleri.github.io/humblefoolcup/)

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+    padding: 0.2em 0;
+}
+
+ 
+   
+p {
+    line-height: 1.33em;
+    color: #7E7E7E;
+}
+h1 {
```
Comment (magician03) [inline css/main.css:15]: @aswanthkoleri ,What is this needed for, it is overwriting the color of headers in index.html. Add class to your required DOM elements and style those by specifying the class. Cross check for other tags and specifiers too.

**Comment by magician03** [comment]
@aswanthkoleri , your stylesheet is overwriting the styles of existing DOM, mentioned the same in inline review.
Name your files as `loader.css` and `loader.js`. It would be more clear and consistent that way

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/*
```
Comment (gauthamzz) [inline js/main.js:1]: please remove this part

**Review by gauthamzz — CHANGES_REQUESTED** [review]

**Comment by magician03** [comment]
Duplicate PR of #52 , closing

**Pull request closed without merging**